### PR TITLE
Add new a native: SetVehicleHeadlightsColor

### PR DESCRIPTION
**Pull request by PichotM**
TITLE

### CODE REVIEW

File: VEHICLE/SetVehicleHeadlightsColor.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ ns: VEHICLE
 ## SET_VEHICLE_HEADLIGHTS_COLOR
 
 ```c
-// 0xE41033B25D003A07
+// _0xE41033B25D003A07
```
Comment (TomGrobbe) [inline VEHICLE/SetVehicleHeadlightsColor.md]: I don't think that's what he meant, he meant the custom name you've given it should be prefixed with `_`. The hash there shouldn't.

Not sure, but I think this is also required (put it below the ns: VEHICLE line, but above the --- ) 
```
aliases: ["0xE41033B25D003A07"]
```

Comment (PichotM) [reply VEHICLE/SetVehicleHeadlightsColor.md]: ty

**Comment by TomGrobbe** [comment]
Just noticed something, might be worth changing it to `...Colour` instead of `Color` since Rockstar tends to use `Colour` instead of `Color` in their native names. 🤷‍♂️

**Comment by PichotM** [comment]
yep, why not do you wanna also add the Get native?

**Pull request closed without merging**